routes/post.go

```diff
@@ -1266,6 +1266,10 @@ func (fes *APIServer) GetSinglePost(ww http.ResponseWriter, req *http.Request) {
 	})
 
 	commentEntryResponseLength := uint32(len(commentEntryResponseList))
+	// If a client provides a default(0) or -ve limit, default to 10
+	if requestData.CommentLimit <= 0 {
```

Comment:
`CommentLimit` is a uint32, so it will never be less than 0. Additionally, this prevents the API from only fetching the post without comments. Users should specify CommentLimit on this endpoint to fetch the number of comments they want.

Reply:
I see. this makes sense!